docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,192 @@
+|`eventType` |String a|Event types displayed in the timeline, which can be:
+
+* `all`: all events
+* `raw`: raw events only
+* `signal`: signals only
+
+|`favorite` |<<favorite-obj, favorite[]>> |Indicates who marked a timeline as a
+favorite, and at what time.
+|`filters` |<<filters-obj, filters[]>> |Filters used in addition to the
+dropzone query. ???
+|`globalNotes` |<<globalNotes-obj, Object[]>> |Notes added to the timeline.
+|`kqlMode` |String a|Indicates whether the KQL bar filters the dropzone query
+results or searches for additional results, where:
+
+* `filter`: filters dropzone query results
+* `search`: displays additional search results
+
+|`kqlQuery` |<<kqlQuery-obj, kqlQuery>> |Object describing the KQL bar query.
+|`pinnedEventIds` |pinnedEventIds[] |Object containing the IDs of pinned
+events.
+|`savedObjectId` |String |The timeline's saved object ID.
+|`savedQueryId` |String |If used, the saved query ID used to filter or search
+dropzone query results.
+|`sort` |sort a|Object indicating how rows are sorted in the timeline's grid:
+
+* `columnId` (string): The ID of the column used to sort results.
+* `sortDirection` (string): The sort direction, which can be either `desc` or
+`asc`.
+
+|`status` |String |: ???
+|`templateTimelineId` |String |: ???
+|`templateTimelineVersion` |Integer |: ???
+|`timelineType` |String a|Indicates whether the timeline is a template or not,
+where:
+
+* `default`: Indicates a timeline used to actively investigate events.
+* `template`: Indicates a timeline template used with detection rules for
+displaying alerts in Timeline.
+
+|`title` |String |The timeline's title.
+|`updated` |Float |The last time the timeline was updated, using a
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the timeline.
+|`version` |String |The timeline's version.
+|==============================================
+
+[[col-obj]]
+[discrete]
+==== columns object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`aggregatable` |Boolean |Indicates whether the field can be aggregated on all
+indices.
+|`category` |String |The ECS field set to which the field belongs.
+|`description` |String |Column field description.
+|`example` |String |Column field example.
+|`indexes` |String |The list of indices where this field has the same type, or
+null if all indices have the same type for the field.
+|`id` |String |ECS field alias or user defined field name.
+|`name` |String |Reserved for future use, typically `null`.
+|`searchable` |Boolean |Indicates whether this field is indexed for search on
+all indices.
+|`type` |String |???
+|==============================================
+
+[[dataProvider-obj]]
+[discrete]
+==== dataProviders object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`id` |String |???
+|`name` |String |???
+|`enabled` |Boolean |???
+|`excluded` |Boolean |???
```

Comment:
whether the filter is `Exclude result` or not.
This is what happen if you exclude a filter `@timestamp exists`
<img width="512" alt="Screenshot 2020-06-25 at 11 39 45" src="https://user-images.githubusercontent.com/6295984/85705118-dae73c80-b6d8-11ea-8bc8-eafba81ae0ed.png">